src/main/java/com/powsybl/network/conversion/server/NotificationService.java

```diff
@@ -31,10 +31,18 @@ public class NotificationService {
     public static final String HEADER_REPORT_UUID = "reportUuid";
     public static final String HEADER_NETWORK_ID = "networkId";
     public static final String HEADER_NETWORK_UUID = "networkUuid";
+    public static final String HEADER_CASE_UUID = "caseUuid";
     public static final String HEADER_RECEIVER = "receiver";
     public static final String HEADER_IMPORT_PARAMETERS = "importParameters";
-    public static final String HEADER_CASE_FORMAT = "caseFormat";
-    public static final String HEADER_CASE_NAME = "caseName";
+    public static final String HEADER_EXPORT_PARAMETERS = "importParameters";
+    public static final String HEADER_FORMAT = "format";
+    public static final String HEADER_FILE_NAME = "fileName";
+    public static final String HEADER_STUDY_UUID = "studyUuid";
```

Comment:
All the uuid below are some gridsuite notions. Powsybl shouldn't know about it. See https://github.com/powsybl/powsybl-network-conversion-server/pull/201/files#r2435189171 to remove them